cluster.go

```diff
@@ -47,32 +47,34 @@ import (
 
 type mongoCluster struct {
 	sync.RWMutex
-	serverSynced  sync.Cond
```

Comment:
I did apply gofmt, it appears that we dont have compile time build check to fail the build. I think its worth adding gofmt check to the build.